internal/services/ingestor/ingestor_v1.go

```diff
@@ -75,6 +76,26 @@ func (i *IngestorImpl) ingestSingleton(tenantId, key string, data []byte, metada
 
 	now := time.Now().UTC()
 
+	go func() error {
+		olapEvent, err := i.repov1.OLAP().CreateEvent(
+			context.Background(),
+			tenantId,
+			sqlcv1.CreateEventParams{
+				Tenantid:           sqlchelpers.UUIDFromStr(tenantId),
+				Generatedat:        sqlchelpers.TimestamptzFromTime(now),
+				Key:                key,
+				Payload:            data,
+				AdditionalMetadata: metadata,
+			},
+		)
+
+		if err != nil || olapEvent == nil {
+			return fmt.Errorf("could not create event in OLAP: %v", err)
+		}
+
+		return nil
+	}()
```

Comment:
I'm a Golang noob, but was thinking that doing this in a Goroutine so it's non-blocking would make sense? not sure if this is a good idea or not